test/server/listener_manager_impl_test.cc

```diff
@@ -749,12 +757,28 @@ version_info: version2
         nanos: 2000000
 )EOF");
 
+  // Validate that workers_started stat is zero before calling startWorkers.
+  EXPECT_EQ(0, server_.stats_store_
+                   .gauge("listener_manager.workers_started", Stats::Gauge::ImportMode::NeverImport)
+                   .value());
+
   // Start workers.
   EXPECT_CALL(*worker_, addListener(_, _));
   EXPECT_CALL(*worker_, start(_));
   manager_->startWorkers(guard_dog_);
+  // Validate that workers_started stat is still zero before workers intimate the status via
```

Comment:
nit: intimate should be initialize

Reply:
@zuercher Thanks for the review. I meant actually "initimate" only here -  I meant to say when workers initimate the listener initialization status via completion call back - then only this stat is updated. If It is not clear, I can update or else please suggest the better wording here. I will update

Reply:
Maybe just "set" then?

Reply:
sure. Changed it. PTAL